es-vue-base/src/lib-components/EsCollapse.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+    <div>
+        <EsButton
+            block
+            :aria-label="id"
+            class="collapse-holder text-left p-0 mb-3 font-weight-bold text-black d-flex align-items-center justify-content-between text-decoration-none text-body"
+            variant="link"
+            @click="isCollapsed = !isCollapsed">
+            <div>
+                <slot name="title" />
+            </div>
+            <div>
+                <IconChevronDown
+                    v-if="isCollapsed"
+                    width="30px"
```

Comment:
Should this size be relative to the inherited font-size vs. hard-coded?

Reply:
How would we accomplish that?